teemo.js

```diff
@@ -1,3 +1,5 @@
+import { CountUp } from './countup.js/dist/countUp.js';
```

Comment:
Include a copy of `counUp.min.js` from https://github.com/inorganik/countUp.js/tree/master/dist in a `js/` directory. Source `./js/countUp.min.js`.